src/Stack/BaseQuery.php

```diff
@@ -604,6 +628,7 @@ public function includeOwner()
      * @return Query
      * */
     public function addParam($key = '', $value = '') 
+    public function addParam($key = '', $value = '')
```

Comment:
@sunil-lakshman Please check here looks like we have duplicate.

Reply:
Removed Duplicate line @uttamukkoji